test/suites/drift/suite_test.go

```diff
@@ -173,7 +188,7 @@ var _ = Describe("Beta/Drift", Label("AWS"), func() {
 		By("creating new security group")
 		createSecurityGroup := &ec2.CreateSecurityGroupInput{
 			GroupName:   awssdk.String("security-group-drift"),
-			Description: awssdk.String("End-to-end Drift Test, should delete after drift test is completed"),
+			Description: awssdk.String("End-to-end Drift Test, should delete after drift coretest is completed"),
```

Comment:
must. find. all. and. replace.
```suggestion
			Description: awssdk.String("End-to-end Drift Test, should delete after drift test is completed"),
```